packages/charts/examples/components/demo/advanced-usage/legend/legend-metadata/legend-metadata-example.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="nui-chart-layout">
+    <div class="chart">
+        <nui-chart class="w-100" [chart]="chart"></nui-chart>
+    </div>
+    <div class="legend">
+        <nui-legend seriesUnitLabel="Mbps" i18n-seriesUnitLabel
+                    [active]="chartAssist.isLegendActive"
+                    [interactive]="true"
+                    (mouseleave)="chartAssist.resetVisibleSeries()">
+            <nui-legend-series [interactive]="false"
+                               [descriptionPrimary]="avgSeries['name']">
+                <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(avgSeries, 'y', undefined, 'y1')"
+                                      backgroundColor="black"
+                                      color="white">
+                </nui-rich-legend-tile>
+            </nui-legend-series>
+            <nui-legend-series *ngFor="let legendSeries of chartAssist.legendSeriesSet; trackBy: chartAssist.seriesTrackByFn"
+                               [descriptionPrimary]="legendSeries['name']"
+                               (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
+                               [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
+                               [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
+                               (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)">
+
+                <!-- Specify 'undefined' for the formatter key since we're not applying any special formatting to the data values shown in the legend -->
+                <!-- Specify the 'y1' data accessor to let the chart assist know how to access the data for the legend. -->
```

Comment:
Maybe these comments could be moved to just above line 12 where the usage is encountered on the page for the first time.